docs/standard-library/built-in-decorators.md

```diff
@@ -8,7 +8,9 @@ toc_max_heading_level: 3
 
 ### `@deprecated` {#@deprecated}
 
-Mark this type as deprecated
+Mark this type as deprecated.
+
+NOTE: It is preferable to the `#deprecated` directive instead.
```

Comment:
Think we should just say don't use that because here otherwise it will carry over the data and have unexpected consequences

Reply:
Should I fully deprecate `@deprecated` with `#deprecated`?  Or just say in the documentation "don't use this"?

Reply:
Decided to just make it more clear in the docs that `@deprecated` should not be used.  We can do the real deprecation as a separate change after discussing it with DPG crew.